certificate/views.py

```diff
@@ -1135,17 +1127,22 @@ def osdag_workshop_download(request):
                 user = user[0]
         name = user.name
         purpose = user.purpose
-        year = '16'
+        year = user.ws_year[2:]
         id =  int(user.id)
         hexa = hex(id).replace('0x','').zfill(6).upper()
         serial_no = '{0}{1}{2}{3}'.format(purpose, year, hexa, type)
         serial_key = (hashlib.sha1(serial_no)).hexdigest()
         file_name = '{0}{1}'.format(email,id)
         file_name = file_name.replace('.', '')
+        details = {
+        'name': name, 'year': user.ws_year,
```

Comment:
I think here if you add indentation then will be better.

Reply:
done